YOUR_ADMIN/zca_bootstrap_colors.php

```diff
@@ -227,7 +227,7 @@
             <?php echo '</form>'; ?>
           </div>
           <div class="row text-right">
-            <a class="btn btn-primary" role="button" href="<?= zen_href_link(FILENAME_ZCA_BOOTSTRAP_COLORS, 'downloadcsv=1', 'SSL') ?>"><?= BUTTON_DOWNLOAD_CSV; ?></a>
+            <a class="btn btn-primary" role="button" href="<? echo zen_href_link(FILENAME_ZCA_BOOTSTRAP_COLORS, 'downloadcsv=1', 'SSL') ?>"><? echo BUTTON_DOWNLOAD_CSV; ?></a>
```

Comment:
Oops. missed the `php` parts in `<?php echo`

Reply:
Yep. Fixed.